docs/web-traffic.md

```diff
@@ -26,4 +26,8 @@ View counts from Documentation Insights data should be considered as approximate
 
 #### Sources
 
-Pixel-based telemetry will work on standard webpages, rendered markdown documentation on package registry sites like `npm` and `PyPi`, and anywhere an image can be embedded, with a notable exception being GitHub. When GitHub renders markdown, it rewrites URLs from their original address to https://camo.githubusercontent.com/<path>, where GitHub hosts any linked images themselves. This cuts out the original host of that content from the equation, and therefore prevents Scarf from providing insights to maintainers.
+Pixel-based telemetry will work on standard webpages, rendered markdown documentation on package registry sites like `npm` and `PyPi`, and anywhere an image can be embedded, with a notable exception being GitHub, because of how GitHub manages page rendering and image serving for your README pages. When GitHub renders markdown, it rewrites URLs from their original address to https://camo.githubusercontent.com/<path>, and hosts the actual images you've embedded on their backend. Thus, when it's rendered by GitHub in a browser,  the browser never makes a request to somesite.com/image.png.  So somesite.com can never see the real traffic coming through, they only see GitHub coming to get the image
+ 
+ Let's say I want to embed an image in my GitHub README. I'll add an embed of the URL, say somesite.com/image.png. That URL will be saved in the source for my README, but when GitHub actually goes to render it in a browser,  my browser will never actually make a request to somesite.com/image.png. Instead, GitHub will fetch somesite.com/image.png on their backend, and generate a new link to that image from their server. It will then render the README, with somesite.com/image.png replaced with camo.githubusercontent.com/<token>. So somesite.com will never see the real traffic coming through, they can only see it as undifferentiated traffic from GitHub.
```

Comment:
Wondering if maybe we should cut this paragraph? This _really_ goes into the details of a limitation but we've already explained it above. Also the first-person aspect of this jumps out at me, if we do indeed want to keep it.

Reply:
I have assigned this P/R to @havi